site/profile/manifests/windows/baseline.pp

```diff
@@ -1,14 +1,13 @@
-class profile::windows::baseline {
+class profile::baseline {
 
-  class { 'chocolatey':
-    notify => Reboot['afterchocolatey'],
+  exec { 'rename-admin':
```

Comment:
I'd like to keep a common baseline. If I remove the rename, are the other changes acceptable?